### PR TITLE
Update EE's JS for jQuery 3

### PR DESCRIPTION
**Pull request by Pebblo**
See #24 

The latest version of jQuery used in 5.6 broke the MER links (the ajax request would throw an error).

So this PR fixes MER , updates some of the JS used in the admin to remove some notices and updates some of the libraries we use. Any of the sections that used ajax in EE3 were broken from [THIS](https://github.com/eventespresso/event-espresso-legacy/commit/a8939bc3aced16bef83ffceef371175754fcbac4), for example re-ordering the Registration form Questions/Groups.

To test use WP 5.6: https://en-gb.wordpress.org/plugins/wordpress-beta-tester/
(I recommend a clean site for this)

1. Use MER and try adding an event to the cart using master. You should see a spinning icon.

Then switch to this branch, hard refresh your browser (CTRL+F5 for me) and try again, this time MER should work as expected.

---

2. Confirm that the event table works as normal and that selecting columns to hide/show still works as expected.

---

3. In Event Espresso -> Questions.

Drag and drop some questions around, check your browsers consoles for error (not deprecated notices, those are fine).

Refresh and check the order 'sticks'.

Repeat with Question Groups.

---

4. Confirm that the promotions admin lists the promotions in Event Espresso -> Promotions.

### CODE REVIEW

**Comment by Pebblo** [comment]
Note, I reverted the commit to update the Datatables and ColVis library as it broke the Promotions admin, so for now I'm leaving it as is.

File: scripts/espresso_EEGlobals_functions.js [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-    jQuery(window).unload( function() {}); 
+    jQuery(window).on("unload", function() {});
 
     jQuery.ajaxSetup({
         cache: false,
         xhr: function()
         {
-            if (jQuery.browser.msie)
-            {
-                return new ActiveXObject("Microsoft.XMLHTTP");
```
Comment (tn3rb) [inline scripts/espresso_EEGlobals_functions.js:12]: noice

**Review by tn3rb — APPROVED** [review]
`jquery.jqplot.min.js` was used for graphs i believe... have you verified that this is not used anywhere? I'm going to assume you have.

Other than the above, everything looks great 👍

**Comment by sethshoultes** [comment]
I've tested these updates locally and all seems to be working as expected.

**Comment by Pebblo** [comment]
@tn3rb I thought I'd updated jqPlot but from the above it sounds like your under the impression I've deleted it and if thats the case thats not what I meant to do, have I not updated it?

You did reminde me to add some details on it though and I have checked the chart loads.

Event Espresso -> Events

In the actions column, click the Graphs button and check it loads.

**Comment by sethshoultes** [comment]
I tested jqPlot and that's working as expected as well.

**Comment by Pebblo** [comment]
Thanks Seth

**Comment by tn3rb** [comment]
> @tn3rb I thought I'd updated jqPlot but from the above it sounds like your under the impression I've deleted it and if thats the case thats not what I meant to do, have I not updated it?

oh ya my bad... looked like the file was deleted. Nothing to see here! move along!

**Pull request merged**